src/sentry/models/group.py

```diff
@@ -238,6 +238,22 @@ def get_oldest_or_latest_event_for_environments(
     return None
 
 
+def get_replay_count_for_group(group: Group) -> int:
+    events = eventstore.backend.get_events(
+        filter=eventstore.Filter(
+            conditions=[["replayId", "!=", None]],
```

Comment:
i think it needs to be empty string `""` instead of `None`?

Reply:
maybe its aliased somewhere and i see the test passes, so this is probably fine. we can create an interface to use the same replay_count query we use on our endpoint if that's deemed necessary and this is too flakey.